### PR TITLE
New metric (Max Depth AST)

### PR DESCRIPTION
**Pull request by silverCase**
Issue #169

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#201](https://github.com/yegor256/aibolit/issues/201) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/2017b686-aece-49e6-a805-06aafea58a8d); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by acheshkov** [comment]
@0crat out

File: aibolit/metrics/mda/mda.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from aibolit.utils.ast import AST
+
+
+class MDAMetric:
+    @staticmethod
+    def get_depth(tree):
```
Comment (acheshkov) [inline aibolit/metrics/mda/mda.py:6]: add typings

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/201#issuecomment-612080582))

@acheshkov GitHub issue is still open, won't close

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job audit: performer @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)[/z](https://www.0crat.com/u/fanifieiev) didn't make CR comments

**Pull request merged**